src/platform/site-wide/user-nav/reducers/index.js

```diff
@@ -25,16 +22,10 @@ function closeAllMenus(menuState) {
   });
 }
 
-function loginStuff(state = initialState, action) {
+export default function userNavReducer(state = initialState, action) {
   switch (action.type) {
-    case UPDATE_LOGGEDIN_STATUS:
-      return _.set('currentlyLoggedIn', action.value, state);
-
     case TOGGLE_LOGIN_MODAL:
-      return _.set('showModal', action.isOpen, state);
-
-    case LOG_OUT:
-      return _.set('currentlyLoggedIn', false, state);
+      return _.set('showLoginModal', action.isOpen, state);
 
     case UPDATE_SEARCH_HELP_USER_MENU:
```

Comment:
I think that makes sense, but there's some code in one of the components that does something similar and I'm not sure if doing both would cause problems.